Phonebook.Frontend/src/styles.scss

```diff
@@ -142,9 +147,14 @@ app-root {
   min-height: 100%;
 }
 
+.pb-text-align-center {
+  text-align: center;
+}
 
-.pb-float-right {
```

Comment:
@mschwrdtnr you removed that class. It is causing issues in the table-settings-dialog: 
![image](https://user-images.githubusercontent.com/13590797/67496823-4a55e980-f67d-11e9-8000-711216220d7c.png)

Reply:
Can you readd it?

Reply:
I didn't see it either :)